configs/11.0/packages/gcc/stage_optimized

```diff
@@ -93,10 +93,15 @@ atcfg_make() {
 	# illegal instruction error, forcing the script to assume that TLS is
 	# not available.
 	#
+	# ssp_have_usable_vsnprintf=define is required to avoid configure
+	# scripts from running programs compiled with a -mcpu that is superior
+	# of the base value, which can cause a silent illegal instruction
+	# error, forcing the script to assume that vsnprintf is not usable.
 	PATH=${at_dest}/bin:${PATH} \
 	libgo_cv_lib_setcontext_clobbers_tls=no \
 		${SUB_MAKE} all \
-			    gcc_cv_have_tls=yes
+			    gcc_cv_have_tls=yes \
+			    ssp_have_usable_vsnprintf=define
```

Comment:
It's fine to keep these variables here, but is there a reason to not keep them together with the other cached variable (libgo_cv_lib_setcontext_clobbers_tls)?

Reply:
AFAIK, variables defined as env vars are not passed to subshells unless they're explicitly exported or if the underlying shell receives them.
I have already confirmed that ssp_have_usable_vsnprintf is affected by this.